tripal_chado/src/Plugin/Field/FieldType/ChadoPropertyTypeDefault.php

```diff
@@ -215,11 +217,43 @@ public static function storageSettingsFormValidate(array $form, FormStateInterfa
     $chado = \Drupal::service('tripal_chado.database');
     $schema = $chado->schema();
     if ($schema->tableExists($prop_table)) {
-      $form_state->setValue(['settings', 'storage_plugin_settings', 'prop_table'], $prop_table);
+      $drupal_10_2 = $form_state->getValue(['field_storage']);
+      if ($drupal_10_2) {
+        $form_state->setValue(['field_storage', 'subform', 'settings', 'storage_plugin_settings', 'prop_table'], $prop_table);
```

Comment:
Again, why not use getFormStateSettings here?

Reply:
I'll have to defer to @dsenalik to answer that question. I don't see why it couldn't but there's that check from drupal_10_2 which I don't understand and I suspect it may be tied to that?

Reply:
It can't be used here because we are setting a value, not getting it 🙃
I had to add a similar thing in the ajax callbacks with commit https://github.com/tripal/tripal/pull/1805/commits/b0dfc9d43253284786d306a75c64a1318cb35d27
Maybe in another PR I could make a `setFormStateSettings()` - but there's enough in this PR